src/utils/googleTagManager.ts

```diff
@@ -137,7 +141,7 @@ export const usePageTracking = (): void => {
     return () => {
       unsubscribe()
     }
-  }, [chainId])
+  }, [chainId, origin])
```

Comment:
Curious, what's origin and why it's needed?

Reply:
Its the host part of the URL with the protocol prefixed since we want to build the location parameter ourselves

Reply:
Why not use `useLocation().pathname` as a dependency instead of listening to the history object?

Reply:
> Its the host part of the URL with the protocol prefixed since we want to build the location parameter ourselves

AFAIU, it's not a React variable, is it? So no need to make it a dependency.

Reply:
Good point, I removed it from the dependency array.